File: WEEK_2/scores.c
#include <cs50.h>
#include <stdio.h>

const int N = 3;

float avarage(int array[], int length);

int main(void)
{
    int scores[N];

    for (int i = 0; i < N; i++)
    {
        scores[i] = get_int("Scores: ");
    }

    printf("Avarage: %f\n", avarage(scores, N));
}

float avarage(int array[], int length)
{
    int sum = 0;

    for (int k = 0; k < length; k++)
    {
        sum += array[k];
    }

    return sum / (float) length;
}

File: WEEK_6/scores.py
def main():

    scores = []

    for i in range(3):
        score = get_int("Number: ")
        scores.append(score)

    average = sum(scores) / len(scores)
    print(f"Average: {average}")


def get_int(prompt):
    while True:
        try:
            return int(input(prompt))
        except ValueError:
            print("Not an integer!")


main()
